hydrolib/core/io/structure/models.py

```diff
@@ -60,6 +64,8 @@ class Comments(INIBasedModel.Comments):
     name: str = Field("id", alias="name")
     type: str = Field(alias="type")
 
+    polylinefile: Optional[Path] = Field(None, alias="polylinefile")
```

Comment:
Should this be an `Optional[DiskOnlyFileModel]`? Or `Optional[PolyFile]`?

Reply:
I have changed it to a DiskOnlyFile model for now as discussed with you @arthurvd.
Changing it to a PolyFile requires the referenced file to exist. This leads a failing test and we want to look further into how to make it configurable so that we can switch it on or off.